src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");
```

Comment:
The message should be more informative and indicate the specific type of Machine. For example, 'Truck started its work' could be changed to 'Truck is working.' to align with the checklist requirement.